wic/src/components/FoodLocations.js

```diff
@@ -1,48 +1,43 @@
 import React, {PropTypes, Component} from 'react';
 import ReactDOM from 'react-dom';
-
-import shouldPureComponentUpdate from 'react';
-
-import Gmap from './Gmap';
-
+// import shouldPureComponentUpdate from 'react';
+import ResoruceMap from './ResoruceMap';
 import ResourceTable from './ResourceTable';
 
-const markers = [{
-    position: {
-      lat: 25.0112183,
-      lng: 121.52067570000001,
-    },
-    key: `Taiwan`,
-    defaultAnimation: 2,
-}];
-
 class FoodLocations extends React.Component {
     constructor(props) {
         super(props);
-        this.state = {
-            apiResultData: ''
-        };
+        this.state = {};
     }
 
-    componentWillReceiveProps() {
-      console.log('componentWillReceiveProps');
-      var self = this;
-      fetch('https://api.smc-connect.org/search?category=Food Pantry&location=' + this.props.zipcode).then(function (response) {
+    findFoodLocations() {
+      const self = this;
```

Comment:
we're using arrow functions, so, "this" should be preserved - but...maybe that didn't work?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions

Reply:
yes, already tried that, react class instance seem to not be in scope inside the promise chain.

Reply:
Oh, that's unfortunate :-(